GovUkDesignSystem/GovUkHtmlHelperExtensions.cs

```diff
@@ -124,6 +124,29 @@ public static class GovUkHtmlHelperExtensions
                 idPrefix);
         }
 
+        public static async Task<IHtmlContent> GovUkCheckboxFromBoolFor<TModel>(
+            this IHtmlHelper<TModel> htmlHelper,
+            Expression<Func<TModel, bool>> propertyLambdaExpression,
+            LabelViewModel label,
```

Comment:
Sorry, only just saw, this should have a default null value too

Reply:
Done